src/ImpactX.cpp

```diff
@@ -252,6 +252,13 @@ namespace impactx
                 pp_element.get("g", g);
                 pp_element.get("K2", K2);
                 m_lattice.emplace_back( DipEdge(psi, rc, g, K2) );
+            } else if (element_type == "constf") {
+                amrex::Real ds, kx, ky, kt;
+                pp_element.get("ds", ds);
```

Comment:
Please don't forget to add those to `docs/source/usage/parameters.rst`

Reply:
Done.  (See above.)